frameworks/cassandra/tests/test_overlay.py

```diff
@@ -43,6 +45,7 @@ def teardown_module(module):
 @pytest.mark.sanity
 @pytest.mark.smoke
 @pytest.mark.overlay
+@pytest.mark.skipif('os.environ.get("SECURITY") == "strict"')
```

Comment:
Why are you using the odd, string-based syntax instead of simply entering this as python? I.E. just remove the outside single quotes